rllib/core/models/catalog.py

```diff
@@ -98,22 +98,33 @@ def __init__(
         self.model_config_dict = {**MODEL_DEFAULTS, **model_config_dict}
         self.view_requirements = view_requirements
 
-        # Produce a basic encoder config.
+        self.latent_dims = None
```

Comment:
If you want to allow people to set latent_dims from outside, I rather make it explicit by creating a setter / getter method for it via properties.

Reply:
Got it, changed 🙂